modules/ROOT/pages/dataweave-formats-java.adoc

```diff
@@ -83,6 +83,17 @@ The Java writer uses the metadata property to discover the Java class to which t
 The Java format supports the `Enum` custom type. To put a Java `enum` value into a `java.util.Map`, the DataWeave Java module defines a custom type called `Enum`.
 This custom type enables you to handle a given string as the name of a specified `enum` type. You must use the `class` metadata property when specifying the Java class name of the enum, for example: `{gender: "Female" as Enum {class: "org.mycompany.Gender"}}`
 
+[[working_with_input_stream]]
```

Comment:
```suggestion
[[working-with-input-stream]]
```

Reply:
seems we should avoid using underscores for anchors
https://confluence.internal.salesforce.com/display/MTDT/File+and+Anchor+Naming+Conventions#:~:text=As%20with%20file%20names%2C%20use%20only%20lowercase%20letters%20and%20hyphens%20(%2D)%20in%20anchor%20names.%20Anchor%20names%20can%27t%20begin%20with%20a%20number.